code/game/world.dm

```diff
@@ -385,6 +389,22 @@ GLOBAL_VAR(restart_counter)
 /world/proc/on_tickrate_change()
 	SStimer?.reset_buckets()
 
+/world/proc/init_byond_tracy()
+	var/library
+
+	switch (system_type)
+		if (MS_WINDOWS)
+			library = "prof.dll"
+		if (UNIX)
+			library = "libprof.so"
+		else
+			CRASH("Unsupported platform: [system_type]")
+
+	var/init_result = call(library, "init")()
```

Comment:
@AnturK Gonna need to be call_ext if your 515 branch gets merged before this PR

Reply:
Mhm

Reply:
I meant my comment the other way around but you know what I mean